ios/keyman/Keyman/Keyman/Classes/View/KMUnderscoreView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  KMUnderscoreView.swift
+//  Keyman
+//
+//  Created by Jacob Bullock on 4/26/18.
+//  Copyright © 2018 SIL International. All rights reserved.
+//
+
+import Foundation
+import  UIKit
+
+class KMUnderscoreView: UIView  {
+    required public init?(coder: NSCoder) {
+        super.init(coder: coder)
+        self.commonInit()
+    }
+
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.commonInit()
+    }
+
+    func commonInit() {
+    }
+
+    override func draw(_ rect: CGRect) {
+        let color: UIColor = .yellow
+
+       
+        let widths: [CGFloat] = [0.6, 0.25, 0.15]
```

Comment:
Add a comment above this to describe where these values are coming from (both widths and colours)